gms_prod.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Deploy the Galaxy Media Site
+- hosts: gms_webservers
+  remote_user: ubuntu
+  become: true
+  vars_files:
+    - group_vars/VAULT
+    - group_vars/gms.yml
+    - host_vars/site.usegalaxy.org.au.yml
+  pre_tasks:
+    - name: update apt cache
+      ansible.builtin.apt:
+        update_cache: yes
+    - name: install system dependencies
+      ansible.builtin.package:
+        name:
+          - libpq-dev
+  roles:
```

Comment:
You'll also need to run the `common` and `insspb.hostname` roles. Probably also `dj-wasabi.telegraf`.

Reply:
common role needs to run as root ({role: common, become: true, become_user: root})